### PR TITLE
fix(styles): fixes styles issues and adds new content

### PR DESCRIPTION
**Pull request by SachinInnov**

### CODE REVIEW

**Comment by SachinInnov** [comment]
this commits fixes following issues: 

- Page Title of the Introduction > designers:- is not the same as other titles, it should start with Capital case.
-  Components > Overview > All components :- Search is not working - Hide search box till it is integrated
- Foundation > Layout > Typography :- The image is not getting loaded Link 
- [Content- Voice & tone guidelines] Incorrect Tabs
-  Right Navbar 
	- should automatically highlight on page visit for nav items visible
	- should scroll to in page title on load/refresh and highlight that nav link
- In Design it is shown three colors in one row in foundation - use grid with responsive sizes for different screen sizes it should show 3 items for medium and above medium screen sizes.
- Foundation> Interactions Duration values table is not aligned with design
- Component > Overview > Status:- Selected tab should be highlighted as link - update TabsWrapper to Tabs
- Content> Voice and tone :- tabs missing 
- Content > House Rules :- page not found
-  Foundation > Response times > Incorrect heading
- Content> Button Guidelines :- Tabs missing
- “Some Heading” at the page’s top? Getting Started?
- How to use Incomplete [Design Issue] 
- “[placeholder]” text in what’s new section
- Missing tab in content page
- In right navigation, things should get selected on basis of scroll position, like if i am landing on a page, the top item should be active by default and change on scrolling, at least it should be selected. We can ignore automatic selection on scroll.
- Button guideline, spacing issue with correct vs incorrect usage section
- Foundation > Layout > Right Navigation > Grid - Unnecessary nesting for types

File: site/src/pages/components/overview/tabs/status.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Components
+description: Components are the reusable building blocks of our design system. Each component meets a specific interaction or UI need, and has been specifically created to work together to create patterns and intuitive user experiences.
+tabs: ['All components', 'Status']
+---
+
+<div style={{ width: 'var(--spacing-9)' }}>
```
Comment (xlreon) [inline site/src/pages/components/overview/tabs/status.mdx:7]: This search bar should also be hidden

**Comment by xlreon** [comment]
1. **[Content- Voice & tone guidelines] Incorrect Tabs**
 Changes missing but the issue is fixed. Need to do RCA on what fixed the issue.

2. **Right Navbar > should scroll to in page title on load/refresh and highlight that nav link**
 Not working on chrome v94 on mac but working on firefox, safari. Working on chrome v94 on linux.

**Comment by SaniyaGupta** [comment]
1. Colors are shown in one row now, but design is not aligned. There is extra margin. Also hex code and RGB collapsing. Please make it responsive
2. Foundation> Interactions Duration values table tokens are not aligned with design

**Pull request merged**